linalg/operator.hpp

```diff
@@ -309,16 +309,23 @@ class Operator
 
 
 /// Base abstract class for first order time dependent operators.
-/** Operator of the form: (x,t) -> f(x,t), where k = f(x,t) generally solves the
+/** Operator of the form: (x,t) -> k(x,t), where k generally solves the
```

Comment:
`TimeDependentOperator::Mult(x,y)` at time `t` gives you `f(x,t)` right?

Reply:
What is your definition of `F` and `G`?  My interpretation of this class was that `Mult` gives `G(x,t)` (mainly because this is how Petsc defines `ExplicitMult`).  So if `F(x, k, t)` is defined as `k`, then `Mult` does indeed give you `k(x,t)` (or `f(x,t)` in the prior notation).

Reply:
I agree with it viewing it from the perspective of the F = G representation.

Reply:
Thinking about this more, I suppose `TimeDependentOperator::Mult` should return `k` if the underlying `Operator` is defined as a mapping to `k`.  I'm running an idea by the SUNDIALS devs to see if the `ARKStepSolver` can use `TimeDependentOperator::ExplicitMult` when `ARKStepSolver` is looking for `g(y,t)` instead of `M^{-1} g(y,t)`.  If that change can be made, then I'll refactor the new documentation to say 
1. `TimeDependentOperator::Mult` returns the `k` that solves `F(x,k,t) = G(x,t) for a given `x` at time `t`.
2. For solving ODEs of the form `M dy/dt = g(y,t)`, `F(x,k,t)` is defined as `M k` and `G` is defined as `g(y,t)`

Reply:
Change is made: `ARKStepSolver` uses `ExplicitMult` when either a `UseMFEMMassLinearSolver` or `UseSundialsMassLinearSolver` is called